conu/backend/docker/container_parameters.py

```diff
@@ -101,13 +114,44 @@ def create_from_drb(cls, run_builder):
 
         options_dict = vars(args)
 
+        healthcheck = Healthcheck()
+        health_cmd = options_dict.pop("health_cmd")
+        health_interval = options_dict.pop("health_interval")
+        health_timeout = options_dict.pop("health_timeout")
+        health_retries = options_dict.pop("health_retries")
+        no_healthcheck = options_dict.pop("no_healthcheck")
+
+        if health_cmd is not None: healthcheck.test = health_cmd
+        if health_interval is not None: healthcheck.interval = health_interval
+        if health_timeout is not None: healthcheck.timeout = health_timeout
+        if health_retries is not None: healthcheck.retries = health_retries
+        # if healthcheck: healthcheck = Healthcheck()
+        options_dict["healthcheck"] = healthcheck
```

Comment:
To squash the code, you can use default parameter in pop https://docs.python.org/3/library/stdtypes.html#dict.pop, So It would look sth like:
```python
if not options_dict.pop("no_healthcheck", None):
    options_dict["healthcheck"] = Healthcheck(
        test=options_dict.pop("health_cmd", None),
        ...
    )
```